src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import { Comment } from '../../types/Comment';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
```

Comment:
`<ul>` instead `<div>`. Please check you code and improve according this comment